client/components/vertical-nav/item/style.scss

```diff
@@ -1,3 +1,6 @@
+@import 'assets/stylesheets/shared/_colors';
```

Comment:
I was wondering if it would be possible to import one file that contains all variables, functions, mixins and stuff that doesn't produce additional CSS itself. With that in place we could use another Webpack loader to prepend it without any changes to the existing files :)